employees/views.py

```diff
@@ -37,41 +39,80 @@ def query_as_dict(query_set):
 class ReportList(APIView):
     renderer_classes = [renderers.TemplateHTMLRenderer]
     template_name = 'employees/report_list.html'
+    reports_dict = {}
+    project_form = ''
     permission_classes = (
         permissions.IsAuthenticated,
     )
 
     def get_queryset(self):
         return Report.objects.filter(author=self.request.user).order_by('-date', 'project__name')
 
+    def _add_project(self, serializer, project):
+        project.members.add(self.request.user)
+        project.full_clean()
+        project.save()
+        project.refresh_from_db()
```

Comment:
Is it necessary here? I think it should work without refreshing database.

Reply:
Done